README.MD

```diff
@@ -65,6 +65,14 @@ mongo_db_bundle:
         ssl: false
         connectTimeoutMS: 3000 # default null (no timeout)
         readPreference: primaryPreferred # see https://docs.mongodb.com/manual/reference/read-preference/#primary for info
+      
+      ssl_client_name:    # see https://www.php.net/manual/en/mongo.connecting.ssl.php 
```

Comment:
This is referencing the wrong extension, the legacy one. This bundle supports `ext-mongodb`, the newer one.